app/controllers/mappings_controller.rb

```diff
@@ -89,6 +89,9 @@ def find_global
     # This allows finding a mapping without knowing the site first.
     render_error(400) and return unless params[:url].present?
 
+    # Strip leading and trailing whitespace before any processing.
+    params[:url] = params[:url].strip
```

Comment:
Could we do this without modifying `params`? I think we tried hard to avoid that elsewhere in Transition. Otherwise 👍